json/mir-common-json/source/mir_common_json.d

```diff
@@ -49,6 +49,7 @@ auto notifyStop(scope const char[] msg = "stop")
     notify(msg);
 }
 
+@serdeIgnoreUnexpectedKeys
```

Comment:
Could you please clarify why did you add this line (and the same below)?

Reply:
Unlike the old one, the new implementation will throw exceptions in case of keys that hasn't been handled by the structure. The UDAs are used to preserve the old behaviour and do not throw.

Reply:
For example, do not throw exception, that this input isn't used:
```
      "name": "wxetbf 1966",
      "opts": {
        "1": [
          1,
          true
        ]
      }
```